File: src/main/java/no/fintlabs/instance/gateway/model/oppfolgingstjenesten/Inntaksadresse.java
package no.fintlabs.instance.gateway.model.oppfolgingstjenesten;

import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.extern.jackson.Jacksonized;

@Getter
@Jacksonized
@EqualsAndHashCode
@Builder
public class Inntaksadresse {

    private String gateadresse;
    private String postnummer;
    private String poststed;
}

File: src/main/kotlin/no/fintlabs/instance/gateway/model/vigo/Inntaksadresse.kt
package no.fintlabs.instance.gateway.model.vigo

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Inntaksadresse(
    val gateadresse: String? = null,
    val postnummer: String? = null,
    val poststed: String? = null
)
